src/pages/new-vehicle.vue

```
@@ -25,72 +58,205 @@
 </template>
 
 <script lang="ts" setup>
-import { ref } from 'vue';
+import { ref, onMounted, watch } from 'vue';
 import SearchBar from '@/components/Search.vue';
 import VehicleOwnerSummaryPage from '@/components/vehicleData/VehicleOwnerSummary.vue';
 import VehicleSummaryPage from '@/components/vehicleData/VehicleSummary.vue';
 import RegistrationLogsPage from '@/components/vehicleData/RegistrationLogs.vue';
```

ESLint

[error] 'RegistrationLogsPage' is defined but never used.

(@typescript-eslint/no-unused-vars)